integration-tests/utils/integration-helpers.js

```diff
@@ -198,7 +198,11 @@ const enableDataCatalog = async (page, testUrl, token) => {
 }
 
 const clickNavChildAndLoad = async (page, tab) => {
-  await noSpinnersAfter(page, { action: () => click(page, navChild(tab)) })
+  // click triggers a page navigation event
+  await Promise.all([
+    page.waitForNavigation(waitUntilLoadedOrTimeout()),
+    noSpinnersAfter(page, { action: () => click(page, navChild(tab)) })
+  ])
```

Comment:
I don't find function handles navigation but the function name clearly indicates a navigation event.

Reply:
This is used for internal navigation-- clicking on the tabs within a workspace (Analyses, Workflows, etc.). I'm not sure `waitForNavigation` will do anything useful in this case, but as long as the tests continue to pass I assume there is no harm in calling it.

Reply:
`waitForNavigation` is useful for synchronization when loading triggers new network requests.